api/net/Areas/Admin/Controllers/MinisterController.cs

```diff
@@ -0,0 +1,124 @@
+using System.Net;
+using System.Net.Mime;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Areas.Admin.Models.Minister;
+using TNO.API.Models;
+using TNO.DAL.Services;
+using TNO.Keycloak;
+
+namespace TNO.API.Areas.Admin.Controllers;
+
+/// <summary>
+/// MinisterController class, provides minister endpoints for the api.
+/// </summary>
+[ClientRoleAuthorize(ClientRole.Administrator)]
+[ApiController]
+[Area("admin")]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[area]/ministers")]
+[Route("api/[area]/ministers")]
+[Route("v{version:apiVersion}/[area]/ministers")]
+[Route("[area]/ministers")]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Unauthorized)]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Forbidden)]
+public class MinisterController : ControllerBase
```

Comment:
@kylgarmor This controller provides the endpoints for the My Minister administration page.